Docs/source/highlights.rst

```diff
@@ -0,0 +1,57 @@
+.. _highlights:
+
+Science Highlights
+==================
+
+WarpX can be used in many domains of laser-plasma science, plasma physics, accelerator physics and beyond.
+Below, we collect a series of scientific publications that used WarpX.
+Please :ref:`acknowledge WarpX in your works`, so we can find your works.
+
+Is your publication missing? :ref:`Contact us <contact>` or edit this page via a pull request.
+
+Plasma-Based Acceleration
+*************************
+
+Scientific works in laser-plasma and beam-plasma acceleration.
+
+- Wang J, Zeng M, Li D, Wang X, Lu W, Gao J.
+  **Injection induced by coaxial laser interference in laser wakefield accelerators**.
+  *preprint*. under review, 2022.
+  `arXiv:2205.12083 <https://arxiv.org/abs/2205.12083>`__
+
+- Mirani F, Calzolari D, Formenti A, Passoni M.
+  **Superintense laser-driven photon activation analysis**.
+  Nature Communications Physics volume **4**.185, 2021
+  `DOI:10.1038/s42005-021-00685-2 <https://doi.org/10.1038/s42005-021-00685-2>`__
+
```

Comment:
We could also add these two arxiv papers:
https://arxiv.org/abs/2112.03489
https://arxiv.org/abs/2112.12581

Reply:
Awesome, now added these :)